package.json

```diff
@@ -83,6 +83,7 @@
     "js-lnurl": "0.5.1",
     "js-sha256": "^0.9.0",
     "jwt-decode": "^3.1.2",
+    "libphonenumber-js": "^1.10.30",
```

Comment:
shouldn't we use the mobile dataset (instead of min)?

https://gitlab.com/catamphetamine/libphonenumber-js#min-vs-max-vs-mobile-vs-core

it has more validation for mobile phone number